pkg/repository/v1/trigger.go

```diff
@@ -117,6 +117,39 @@ type TriggerFromEventsResult struct {
 	Events []*EventDataForOlap
 }
 
+func (r *sharedRepository) processWorkflowExpression(ctx context.Context, workflow *sqlcv1.ListWorkflowsForEventsRow, opt EventTriggerOpts) (bool, error) {
+	if !workflow.EventExpression.Valid || workflow.EventExpression.String == "" {
+		return true, nil
+	}
+
+	var inputData map[string]interface{}
+	if opt.Data != nil {
+		err := json.Unmarshal(opt.Data, &inputData)
+		if err != nil {
+			return false, fmt.Errorf("failed to unmarshal input data: %w", err)
+		}
+	} else {
+		inputData = make(map[string]interface{})
+	}
+
+	match, err := r.celParser.EvaluateBooleanExpression(
+		workflow.EventExpression.String,
+		inputData,
```

Comment:
In other places we structure our input data to have an `input` and `metadata` field (or something like that), did we double check that this is consistent with that?

Reply:
do you mean in the cel parser or elsewhere in the code? I don't think I did this though, but just checking to make sure

Reply:
Got it! I think I fixed that [here](https://github.com/hatchet-dev/hatchet/pull/1676/commits/3bf14f81c827db46f478c13734105c658d26183e) - lmk if that's closer to what you're thinking

Reply:
yep that looks right!